city_metrix/layers/layer.py

```diff
@@ -52,6 +56,37 @@ def groupby(self, zones, layer=None):
         """
         return LayerGroupBy(self.aggregate, zones, layer, self.masks)
 
+    def write(self, bbox, output_path, tile_degrees=None):
+        """
+        Write the layer to a path. Does not apply masks.
+
+        :param bbox: (min x, min y, max x, max y)
+        :param output_path: local or s3 path to output to
```

Comment:
The `output_path` needs to be a folder name if written to multiple files, but needs to be a file name with a suffix if written to one file. We could either add instructions for this in the notebook or update the code to make it consistent.